src/Our.Umbraco.AuthU/Data/Migrations/UmbracoDbOAuthRefreshTokenStore/Add/AddDeviceIdColumn.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Umbraco.Core.Migrations;
+
+namespace Our.Umbraco.AuthU.Data.Migrations.UmbracoDbOAuthRefreshTokenStore
+{
+    /// <summary>
+    /// Simple migration to create the DeviceId column on the OAuthRefreshToken Table
+    /// </summary>
+    internal class AddDeviceIdColumn : MigrationBase
+	{
+		public AddDeviceIdColumn(IMigrationContext context) : base(context)
+		{ }
+
+        public override void Migrate()
+        {
+            if (!ColumnExists("OAuthRefreshToken", "DeviceId"))
+            {
```

Comment:
For consistency, we should either do away with the curly brackets, or make sure we use curly brackets everywhere. My usual rule is, if it's a one line statement, then don't use them.